test/test_runner.rb

```diff
@@ -68,17 +52,6 @@ def test_local_miss_patch_config
     run_and_assert_error_messages(argv, expects)
   end
 
-  def test_single_mode
```

Comment:
Why did you remove this test case? We don't have specific asserts, yes, but we verify the interactions between Runner and ReportBuilder, DiffBuilder, HtmlReportBuilder, e.g. DiffBuilder needs to be called twice.